FHEM/00_SIGNALduino.pm

```diff
@@ -3164,13 +3165,20 @@ SIGNALduino_Parse_MS($$$$%)
 			#$dmsg = @retvalue;
 			#undef(@retvalue); undef($rcode);
 			
-			
 			my $modulematch = undef;
 			if (defined($ProtocolListSIGNALduino{$id}{modulematch})) {
 				$modulematch = $ProtocolListSIGNALduino{$id}{modulematch};
 			}
 			if (!defined($modulematch) || $dmsg =~ m/$modulematch/) {
 				Debug "$name: dispatching now msg: $dmsg" if ($debug);
+				if (defined($ProtocolListSIGNALduino{$id}{developId}) && substr($ProtocolListSIGNALduino{$id}{developId},0,1) eq "m") {
```

Comment:
Es stehen in $hash->{msIdList} doch ohnehin nur Protokolle, die in developID definiert sind oder?

Reply:
wenn keine Whitelist, Blacklist und developID definiert sind, stehen in der $hash->{msIdList} alle MS-IDs der $ProtocolListSIGNALduino.

Wenn es die Whitelist gibt, dann stehen in der $hash->{msIdList} die IDs der Whitelist.

Wenn es keine Whitelist gibt, dann sind die Protokolle mit developID nicht in der $hash->{msIdList}

Reply:
Hmm, das hatte ich jetzt nicht so verstanden.

Mein Verständnis war: Alles was developID gesetzt hat, steht nur in der msIdList, wenn es auch über ein Attribut gesetzt ist, sonst haben all die Anwender die sich keine Whitelist setzen die ganzen "unfertigen" Protokolle drin.

Reply:
ich habe es hier beschrieben:
https://forum.fhem.de/index.php/topic,58397.msg692568.html#msg692568